Sources/BraveWallet/Crypto/Stores/NFTStore.swift

```diff
@@ -196,10 +196,10 @@ extension NFTStore: BraveWalletKeyringServiceObserver {
   }
   public func autoLockMinutesChanged() {
   }
-  public func selectedAccountChanged(_ coinType: BraveWallet.CoinType) {
-    DispatchQueue.main.async { [self] in
-      update()
-    }
+  public func selectedWalletAccountChanged(_ account: BraveWallet.AccountInfo) {
+  }
```

Comment:
we probably still need the `update()` when selected wallet account is changed.

Reply:
I figured we didn't need these anymore (same change in `PortfolioStore`)? Portfolio & NFT don't care about the currently selected account / account's coin type (all assets are shown regardless of selected account/coin) 🤔.